.github/workflows/Build and test.yml

```
@@ -33,36 +33,28 @@
+        docker run --rm 
+        -v ${{ github.workspace }}:/Nextag-Embedded-Base 
+        -w /Nextag-Embedded-Base/ avr-gcc-toolchain
+          cmake -G Ninja -S . -B build/${{ matrix.build_type }} 
+          -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DNEXTAG_EMBEDDED_PLATFORM_CONFIGURE_AS_UNO=ON
 
     - name: Build
-      run:
-        export PATH="$PWD/avr-toolchain/avr-gcc-13.2.0-x64-linux/bin:$PATH" &&
-        cmake --build ${{github.workspace}}/build --config ${{env.BUILD_TYPE}}
+      run: >
+        docker run --rm 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)